### PR TITLE
Project command tweaks

### PR DESCRIPTION
**Pull request by brandonSc**
* Be more clear about what is being deleted in a project
* Ask for confirmation before deleting the project (unless `-f` flag is provided).
* Can use `-p` flag as alias to `--project`

Screen Shot 2023-03-10 at 2 24 22 PM

### CODE REVIEW

**Review by mikejholly — APPROVED** [review]
Good stuff. I think it makes sense as it's a pretty destructive action. Wonder if we should include a `--force` option as well.

**Comment by brandonSc** [comment]
> Good stuff. I think it makes sense as it's a pretty destructive action. Wonder if we should include a `--force` option as well.

I think that's a good call, in case someone wants to script this.

**Comment by brandonSc** [comment]
> > Good stuff. I think it makes sense as it's a pretty destructive action. Wonder if we should include a `--force` option as well.
> 
> I think that's a good call, in case someone wants to script this.

With `--force` option:

Screen Shot 2023-03-10 at 2 59 51 PM

**Review by nelsam — APPROVED** [review]

File: cmd/earthly/project_cmds.go [hunk 1]
```diff
@@ -124,6 +125,17 @@ func (app *earthlyApp) actionProjectRemove(cliCtx *cli.Context) error {
...
+	answer, err := promptInput(cliCtx.Context,
+		"WARNING: you are about to permanently delete this project and all of its associated pipelines, build history and secrets.\n"+
+			"Would you like to continue?\n"+
+			"Type 'y' or 'yes': ")
+	answer = strings.TrimSpace(strings.ToLower(answer))
+
+	if answer != "y" && answer != "yes" {
+		app.console.Printf("Operation aborted.")
+		return nil
+	}
```
Comment (vladaionescu) [inline cmd/earthly/project_cmds.go:137]: We generally try not to make anything interactive, unless we have no choice. Reason being that it's harder to work with it in automation.

I think it's ok not to have any prompt here, the same way `rm -rf ~` doesn't present a prompt either.

Comment (vladaionescu) [reply cmd/earthly/project_cmds.go:137]: `-f` could be a decent compromise if we want to keep the prompt.

Comment (vladaionescu) [reply cmd/earthly/project_cmds.go:137]: If we keep it, can you make the format of the question consistent with other prompts? e.g. I think we had things like `[yN]` in others.

Comment (vladaionescu) [reply cmd/earthly/project_cmds.go:137]: I was saying "go ahead and always delete the project". But prompt + option to use -f to skip the prompt is also ok.

**Pull request merged**